### ISSUE TITLE
undefined symbol: Pa_Initialize on Ubuntu 12.04

### ISSUE THREAD

**Issue by kommander** [ZECTBynmo/node-core-audio#33]
Getting:
node: symbol lookup error: /home/sebastian/ots0614/node_modules/node-core-audio/build/Release/NodeCoreAudio.node: undefined symbol: Pa_Initialize

Having PA v19 installed.

**Comment by xseignard** [comment]
Hello,
Same problem here, i'm on ubuntu 14.10, I installed portaudio with the following
```
sudo apt-get install libportaudio-dev libportaudio0
```

Any idea?

**Comment by xseignard** [comment]
Any updates?

**Comment by jdfreder** [comment]
I encountered the same problem, it may be the version of port audio... I haven't checked yet though.

**Comment by jdfreder** [comment]
... I should add, I'm running this on a Raspberry PI .

**Comment by jdfreder** [comment]
@xseignard did you ever get this to work?

**Comment by xseignard** [comment]
no, I gave up and used a combination of web audio api + webosckets

**Comment by guicontat** [comment]
Hello, 

I'm on Ubuntu 14.04 and I tried those things : 

Download Port Audio source in ~/node_modules/node-core-audio/portaudio with this link http://www.portaudio.com/archives/pa_stable_v19_20140130.tgz

Then in portaudio folder 

./configure
make install

Then i changed the binding.gyp file in the module folder, especially the linux part

'OS=="linux"', {
	"libraries" : [
		'<(module_root_dir)/portaudio/lib/.libs/libportaudio.so',
		'<(module_root_dir)/portaudio/include/portaudio.h'
	],
	'cflags': [ "-fno-exceptions -lrt -lasound -ljack -lpthread -fPIC" ],
        'cflags!': [ "-fno-exceptions-lrt -lasound -ljack -lpthread -fPIC" ],
	'cflags_cc!': [ "-fno-exceptions -lrt -lasound -ljack -lpthread -fPIC"],
	'cflags_cc': [ "-std=c++0x -lrt -lasound -ljack -lpthread -fPIC" ]
}

And after that i start "node-gyp rebuild" in the module folder

And it seem to work !

**Comment by jdfreder** [comment]
Thanks @guicontat !  I've been using WebAudio over SlimerJS, but I'd like to still give this a try.

**Comment by maximilianmikus** [comment]
thx @guicontat that worked!

**Comment by ZECTBynmo** [comment]
Thanks for working towards Ubuntu support!! Especially @guicontat

Since it appears you guys have gotten it working, would anyone be willing to submit a pull request? :)

Much appreciated!

**Comment by guicontat** [comment]
@ZECTBynmo  Can you add me in the contributors in order to create a new branch please :) ?

**Comment by ZECTBynmo** [comment]
@guicontat why not just fork the repository?

**Comment by renschler** [comment]
I just ran into this issue and like @jdfreder  I am also running this on a Raspberry PI (Rasbian).
